source/speech/__init__.py

```diff
@@ -506,26 +532,33 @@ def getIndentationSpeech(indentation, formatConfig):
 		else: 
 			#we have more than 72 spaces (18 tabs), and must speak it since we don't want to hurt the users ears.
 			speak = True
-	return (" ".join(res) if speak else "")
+	return [" ".join(res)] if speak else []
 
 from .priorities import *
 
-def speak(speechSequence, symbolLevel=None, priority=None):
+
+# C901 'speak' is too complex
+# Note: when working on speak, look for opportunities to simplify
+# and move logic out into smaller helper functions.
+def speak(  # noqa: C901
+		speechSequence: SpeechSequence,
+		symbolLevel: Optional[int] = None,
+		priority: Optional[int] = None

```

Comment:
I'm not sure whether I suggested this before, but as speech priorities are new to speech refactor and thus new to the Python 3 version of NVDA, we could consider making this a enum.IntEnum. Not as part of this pr, though, but I'm happy to provide a pr accordingly.

Reply:
That sounds like a good idea if you have time for it 🥇